### PR TITLE
Add healthcheck endpoint.

### PR DESCRIPTION
**Pull request by sebs-code**
This ticket adds a health check endpoint using the directory_healthcheck package. 

Ticket: https://uktrade.atlassian.net/browse/GREATUK-745

### CODE REVIEW

File: conf/settings.py [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
 # As app is running behind a host-based router supplied by Heroku or other
 # PaaS, we can open ALLOWED_HOSTS
 ALLOWED_HOSTS = ['*']
 
-INSTALLED_APPS = ['revproxy', 'core']
+INSTALLED_APPS = [
+    'revproxy',
+    'core',
+    'directory_healthcheck',
+    'health_check.db',
```
Comment (hal274) [inline conf/settings.py:28]: This app does not have a database. I am not sure why you need these.

Comment (sebs-code) [reply conf/settings.py:28]: The whole thing, or just these imports? 

Even without a database, Don't we still need a healthcheck to see if the app is up / running?

Comment (hal274) [reply conf/settings.py:28]: We need the health check but it does not need to verify database connectivity etc as it does not have one. For this app just a 200 to show it's running successfully should suffice

Comment (sebs-code) [reply conf/settings.py]: Apologies Hal - understood. I have removed the redundant imports (they were meant to be removed). We can remove the health_check.db and health_check.cache ones, but need to keep the directory_healthcheck one for the template. 

Good spot!

**Review by hal274 — APPROVED** [review]

**Pull request merged**